README.md
# Vanilla OS Calamares Settings
This package is based on the original calamares-settings-ubuntu. This is not
meant to be used on any distributions other than Vanilla OS as it will not
handle conflicts with other distributions.

Calamares is just a momentary installer, we will move to the GNOME os-installer
when it is ready and when all the dependencies are available in Ubuntu.

---

Original work by Simon Quigley.

debian/changelog
calamares-settings-ubuntu (100:20.0.0-99pika1.lunar) lunar; urgency=low

  * Weird fix for weird calamares

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:19.0.0-99pika1.lunar) lunar; urgency=low

  * Weird fix for weird calamares

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:19.0.0-99pika.lunar) lunar; urgency=low

  * Weird fix for weird calamares

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:18.0.0-99pika.lunar) lunar; urgency=low

  * Weird fix for weird calamares

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:17.0.0-99pika.lunar) lunar; urgency=low

  * Weird fix for weird calamares

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:16.0.0-99pika.lunar) lunar; urgency=low

  * Weird fix for weird calamares

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:15.0.0-99pika1.lunar) lunar; urgency=low

  * Refind fixes + Booster + hyprland package

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:14.0.0-99pika1.lunar) lunar; urgency=low

  * Refind fixes + Booster + hyprland package

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:13.4.0-99pika8.lunar) lunar; urgency=low

  * hyprland hypr things

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:13.4.0-99pika6.lunar) lunar; urgency=low

  * hyprland hypr things

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200


calamares-settings-ubuntu (100:13.3.0-99pika3.lunar) lunar; urgency=low

  * F*ck keyboards

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:13.3.0-99pika2.lunar) lunar; urgency=low

  * F*ck keyboards

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:13.2.0-99pika1.lunar) lunar; urgency=low

  * Kill me

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:13.1.0-99pika1.lunar) lunar; urgency=low

  * Keyboard fixes for hyprland

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:13.0.0-99pika1.lunar) lunar; urgency=low

  * Refind now part of baseos so remove the apt install commands

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200


calamares-settings-ubuntu (100:12.0.0-99pika2.lunar) lunar; urgency=low

  * typo fix

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:11.0.0-99pika1.lunar) lunar; urgency=low

  * typo fix

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:10.0.0-99pika1.lunar) lunar; urgency=low

  * fcitx5 config fix

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:9.0.0-99pika1.lunar) lunar; urgency=low

  * fcitx5 config fix

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:8.0.0-99pika1.lunar) lunar; urgency=low

  * Refind theme

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200
 
calamares-settings-ubuntu (100:7.98~99pika2.lunar) lunar; urgency=low

  * Default to btrfs properly this time

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:7.96~99pika2.lunar) lunar; urgency=low

  * Default to btrfs properly this time

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:7.95~99pika2.lunar) lunar; urgency=low

  * Smol fix

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200

calamares-settings-ubuntu (100:7.94~99pika1.lunar) lunar; urgency=low

  * Update for lunar

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200


calamares-settings-ubuntu (100:7.93) kinetic; urgency=low

  * First release

 -- Ward Nakchbandi <hotrod.master@hotmail.com>  Sat, 01 Oct 2022 14:50:00 +0200


calamares/branding/vanilla/lang/calamares-vanilla-os_it.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="it_IT">
<context>
    <name>show</name>
    <message>
        <location filename="../show.qml" line="32"/>
        <source>Get Support</source>
        <translation>Ricevi Supporto</translation>
    </message>
    <message>
        <location filename="../show.qml" line="56"/>
        <source>https://github.com/Vanilla-OS</source>
        <translation>https://github.com/Vanilla-OS</translation>
    </message>
    <message>
        <location filename="../show.qml" line="90"/>
        <source>Your Mail</source>
        <translation>La tua Mail</translation>
    </message>
    <message>
        <location filename="../show.qml" line="114"/>
        <source>Managed by Trojitá</source>
        <translation>Gestito da Trojitá</translation>
    </message>
    <message>
        <location filename="../show.qml" line="148"/>
        <source>SMPlayer</source>
        <translation>SMPlayer</translation>
    </message>
    <message>
        <location filename="../show.qml" line="172"/>
        <source>To Play All of Your Movies</source>
        <translation>Per Riprodurre Tutti i Tuoi Filmati</translation>
    </message>
    <message>
        <location filename="../show.qml" line="206"/>
        <source>LibreOffice</source>
        <translation>LibreOffice</translation>
    </message>
    <message>
        <location filename="../show.qml" line="230"/>
        <source>Edits All Of Your Documents</source>
        <translation>Modifica tutti i tuoi documenti</translation>
    </message>
    <message>
        <location filename="../show.qml" line="264"/>
        <source>Pictures</source>
        <translation>Immagini</translation>
    </message>
    <message>
        <location filename="../show.qml" line="288"/>
        <source>Viewed With Nomacs</source>
        <translation>Visualizzato con Nomacs</translation>
    </message>
    <message>
        <location filename="../show.qml" line="322"/>
        <source>Have Fun!</source>
        <translation>Divertiti!</translation>
    </message>
    <message>
        <location filename="../show.qml" line="346"/>
        <source>With the 2048 game</source>
        <translation>Con il gioco 2048</translation>
    </message>
    <message>
        <location filename="../show.qml" line="380"/>
        <source>IRC Chat</source>
        <translation>Chat IRC</translation>
    </message>
    <message>
        <location filename="../show.qml" line="404"/>
        <source>With Quassel IRC</source>
        <translation>Con Quassel IRC</translation>
    </message>
    <message>
        <location filename="../show.qml" line="438"/>
        <source>View Your Documents</source>
        <translation>Vedi i tuoi documenti</translation>
    </message>
    <message>
        <location filename="../show.qml" line="462"/>
        <source>With QPDFView</source>
        <translation>Con QPDFView</translation>
    </message>
</context>
</TS>


calamares/branding/vanilla/lang/calamares-vanilla-os_ru.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="ru_RU">
<context>
    <name>show</name>
    <message>
        <location filename="../show.qml" line="32"/>
        <source>Get Support</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="56"/>
        <source>https://github.com/Vanilla-OS</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="90"/>
        <source>Your Mail</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="114"/>
        <source>Managed by Trojitá</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="148"/>
        <source>SMPlayer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="172"/>
        <source>To Play All of Your Movies</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="206"/>
        <source>LibreOffice</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="230"/>
        <source>Edits All Of Your Documents</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="264"/>
        <source>Pictures</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="288"/>
        <source>Viewed With Nomacs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="322"/>
        <source>Have Fun!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="346"/>
        <source>With the 2048 game</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="380"/>
        <source>IRC Chat</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="404"/>
        <source>With Quassel IRC</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="438"/>
        <source>View Your Documents</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="462"/>
        <source>With QPDFView</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>


vanilla-os/branding/vanilla-os/lang/calamares-vanilla-os_no.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="no">
<context>
    <name>show</name>
    <message>
        <location filename="../show.qml" line="32"/>
        <source>Get Support</source>
        <translation>Få brukerstøtte</translation>
    </message>
    <message>
        <location filename="../show.qml" line="56"/>
        <source>https://github.com/Vanilla-OS</source>
        <translation>https://github.com/Vanilla-OS</translation>
    </message>
    <message>
        <location filename="../show.qml" line="90"/>
        <source>Your Mail</source>
        <translation>Din epost</translation>
    </message>
    <message>
        <location filename="../show.qml" line="114"/>
        <source>Managed by Trojitá</source>
        <translation>Håndtert av Trojitá</translation>
    </message>
    <message>
        <location filename="../show.qml" line="148"/>
        <source>SMPlayer</source>
        <translation>SMPlayer</translation>
    </message>
    <message>
        <location filename="../show.qml" line="172"/>
        <source>To Play All of Your Movies</source>
        <translation>For å spille alle dine filmer</translation>
    </message>
    <message>
        <location filename="../show.qml" line="206"/>
        <source>LibreOffice</source>
        <translation>LibreOffice</translation>
    </message>
    <message>
        <location filename="../show.qml" line="230"/>
        <source>Edits All Of Your Documents</source>
        <translation>Skriv alle dine dokumenter</translation>
    </message>
    <message>
        <location filename="../show.qml" line="264"/>
        <source>Pictures</source>
        <translation>Bilder</translation>
    </message>
    <message>
        <location filename="../show.qml" line="288"/>
        <source>Viewed With Nomacs</source>
        <translation>Vist med Nomacs</translation>
    </message>
    <message>
        <location filename="../show.qml" line="322"/>
        <source>Have Fun!</source>
        <translation>Ha det gøy!</translation>
    </message>
    <message>
        <location filename="../show.qml" line="346"/>
        <source>With the 2048 game</source>
        <translation>Med 2048-spillet</translation>
    </message>
    <message>
        <location filename="../show.qml" line="380"/>
        <source>IRC Chat</source>
        <translation>IRC Chat</translation>
    </message>
    <message>
        <location filename="../show.qml" line="404"/>
        <source>With Quassel IRC</source>
        <translation>Med Quassel IRC</translation>
    </message>
    <message>
        <location filename="../show.qml" line="438"/>
        <source>View Your Documents</source>
        <translation>Se dine dokumenter</translation>
    </message>
    <message>
        <location filename="../show.qml" line="462"/>
        <source>With QPDFView</source>
        <translation>Med QPDFView</translation>
    </message>
</context>
</TS>


calamares/branding/vanilla/lang/calamares-vanilla-os_zh-hans.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="zh_CN">
<context>
    <name>show</name>
    <message>
        <location filename="../show.qml" line="32"/>
        <source>Get Support</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="56"/>
        <source>https://github.com/Vanilla-OS</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="90"/>
        <source>Your Mail</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="114"/>
        <source>Managed by Trojitá</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="148"/>
        <source>SMPlayer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="172"/>
        <source>To Play All of Your Movies</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="206"/>
        <source>LibreOffice</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="230"/>
        <source>Edits All Of Your Documents</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="264"/>
        <source>Pictures</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="288"/>
        <source>Viewed With Nomacs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="322"/>
        <source>Have Fun!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="346"/>
        <source>With the 2048 game</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="380"/>
        <source>IRC Chat</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="404"/>
        <source>With Quassel IRC</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="438"/>
        <source>View Your Documents</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../show.qml" line="462"/>
        <source>With QPDFView</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
